### DISCUSSION TITLE
Linux/WSL: git best practices - use system git or homebrew's?

### DISCUSSION THREAD

**Discussion by juanmirocks** [Homebrew/discussions#4791] [Linux] [answered]
### Additional Info

- [X] I am using Windows Subsystem for Linux

### Output of `brew config`

```shell
Not relevant (no issues; just a question)
```

### Output of `brew doctor`

```shell
Not relevant (no issues; just a question)
```

### Description of issue

I wonder which git package I should use: the system's or homebrew's?

If recommended, I would just use Homebrew's package (and also for other git extensions like `git-lfs`).

However, the [Homebrew on Linux page](https://docs.brew.sh/Homebrew-on-Linux) lists `git` as a system package requirement. Thus I'm in doubt. Are there any best practices for Homebrew on Linux regarding `git`?

**Comment by SMillerDev** [comment, marked as answer]
Do you need new git features that your system doesn't provide? Use Homebrew. Otherwise, use whatever you think is easiest.

**Reply by juanmirocks** [reply]
Fair enough! Thank you for your input.
